### ISSUE TITLE
Running ES service causes issues

### ISSUE THREAD

**Issue by kendaleiv** [elastic/elasticsearch-net-abstractions#11]
On Windows 10 with the `elasticsearch-service-x64` version 6.2.4 running I'm unable to get `Elastic.Xunit.ExampleMinimal` to work as expected. However, with the service stopped it works great.

I'm guessing this is related:

```
Caused by: org.elasticsearch.http.BindHttpException: Failed to bind to [9200]
```

**Comment by Mpdreamz** [comment]
Thanks for raising this @kendaleiv 

By default the nodes `Elastic.Xunit` spin up run on `9200` like any elasticsearch node. This can be changed however e.g if we alter the constructor in the `Elastic.Xunit.ExampleMinimal` `MyTestCluster` cluster.

```csharp
		public MyTestCluster() : base(new XunitClusterConfiguration("6.2.0")
		{
			StartingPortNumber = 9201
		})
		{
		}
```

I'm inclined to keep it this way rather than trying to do something clever, @codebrain @russcam thoughts?

**Comment by codebrain** [comment]
I would be inclined to leave it the way it currently is, otherwise we'd have to start inspecting the local environment to determine available ports and that seems like overkill at this stage.

**Comment by kendaleiv** [comment]
@Mpdreamz Setting `ClusterConfiguration.StartingPortNumber = 9201;` doesn't seem to work and port 9200 is still used. Any ideas?

**Comment by kendaleiv** [comment]
However, setting it like

```csharp
public MyTestCluster() : base(new XunitClusterConfiguration("6.2.0") { StartingPortNumber = 9251 })
```

does work.

**Comment by Mpdreamz** [comment]
This is somewhat expected as everything materializes in the constructor chaining. 

It oughta be a constructor argument really :)


Are you already using any of the code in this repository @kendaleiv ?

**Comment by kendaleiv** [comment]
Yeah, we moved to this from https://github.com/poulfoged/Elasticsearch-Inside. I understand it's prerelease. :smiley:

**Comment by kendaleiv** [comment]
Yeah, we moved to this from https://github.com/poulfoged/Elasticsearch-Inside. I understand this project is prerelease. :smiley:

**Comment by kendaleiv** [comment]
Yeah, we moved to this from https://github.com/poulfoged/Elasticsearch-Inside. I understand this project is currently prerelease. :smiley:
